### PR TITLE
Improve a warning message.

### PR DESCRIPTION
**Pull request by Emeric54**

### CODE REVIEW

File: lib/Bailador/Configuration.pm [hunk 1]
```diff
@@ -134,8 +134,13 @@ class Bailador::Configuration {
                 self.load-from-hash(%config);
                 return;
             }
-            warn 'Error while loading the YAML config file.';
-            warn 'Bailador will use his default configuration.';
+            warn qq:to/WARNING/;
+            Error while loading the YAML config file.
+            Bailador will use his default configuration :
+                Mode : production
```
Comment (ufobat) [inline lib/Bailador/Configuration.pm]: shoudn't we fetch those values from the config object, if possible.
this is redundant information.

Comment (Emeric54) [reply lib/Bailador/Configuration.pm]: yep, it was my initial thought, but I don't know if it's possible to do it easily. Or we can use a new variable....
```perl6
my $mode = 'production';
class Bailador::Configuration {
      has Str $.mode is rw where "production"|"development" = $mode;
}
warn $mode;
```

**Comment by Emeric54** [comment]
I think it's just better to keep it simple. 😀

**Comment by ufobat** [comment]
so basically we can close this, because nothing was changed, right?

**Comment by Emeric54** [comment]
We just now use only one `warn` instruction, the output is better

**Comment by ufobat** [comment]
thanks :)

**Pull request merged**